notifier/src/main/kotlin/modules/MailNotifier.kt

```diff
@@ -19,34 +19,42 @@
 
 package org.ossreviewtoolkit.notifier.modules
 
+import java.nio.charset.Charset
+
 import org.apache.commons.mail.DefaultAuthenticator
-import org.apache.commons.mail.Email
-import org.apache.commons.mail.SimpleEmail
+import org.apache.commons.mail.HtmlEmail
 
 import org.ossreviewtoolkit.model.config.SendMailConfiguration
 
 /**
  * Notification module that provides a configured email client.
  */
 class MailNotifier(config: SendMailConfiguration) {
-    private val client: Email
+    private val client: HtmlEmail = HtmlEmail()
 
     init {
-        client = SimpleEmail()
-
-        client.setHostName(config.hostName)
+        client.hostName = config.hostName
         client.setSmtpPort(config.port)
         client.setAuthenticator(DefaultAuthenticator(config.username, config.password))
-        client.setSSLOnConnect(config.useSsl)
+        client.isSSLOnConnect = config.useSsl
```

Comment:
Nit: Unrelated change?

Reply:
This was not possible with the `Email` interface.
it is only possible with `HtmlEmail`. Should I move it to a separate commit?

Reply:
Interesting that this is only possible with `HtmlEmail`. If you want, you could move this to a succeeding commit with an explanation that this is now possible but wasn't before. But maybe that's overdoing it; I'd also be fine to keep it as-is.